INSTALL.md

```diff
@@ -93,6 +93,20 @@ The [Releases](https://github.com/neovim/neovim/releases) page provides pre-buil
 
 ## Linux
 
+### Pre-built archives
+
+The [Releases](https://github.com/neovim/neovim/releases) page provides pre-built binaries for Linux systems.
```

Comment:
What are the limitations of the tarball? surely it runs on fewer systems than the appimage. Not sure why that isn't specified in the releases page.

By the way, this PR duplicates the notes already on the releases page.

Reply:
Guess it'd depend on what system the tarballs are generated on, is that done via CI?

Reply:
oh I remember now, the tarbar is just a copy of the files from the appimage. sigh.